orchestra/models.py

```diff
@@ -265,7 +268,7 @@ class Status:
         (Status.PROCESSING, 'Processing'),
         (Status.SUBMITTED, 'Submitted'))
 
-    start_datetime = models.DateTimeField(auto_now_add=True)
+    start_datetime = models.DateTimeField(default=timezone.now)
```

Comment:
ah--yeah the default would create them all with a timestamp of whenever you run the migration. I'm not sure what the correct value would even be, here's one proposal:
* migration 1 adds the field, setting all values to the present datetime
* data migration 2 changes all `task.start_datetime` to `task.assignments.first().start_datetime`